sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosClientTelemetryConfig.java

```diff
@@ -30,7 +30,6 @@
 /**
  * Class with config options for Cosmos Client telemetry
  */
-@Beta(value = Beta.SinceVersion.V4_37_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
 public final class CosmosClientTelemetryConfig {
```

Comment:
nit - since beta APIs are gone now, may be remove beta import if no longer used.

Reply:
Good point - by the way when doing this I also went through all the APIs still annotated with Beta. I feel for many of them we should simply remove the Beta tag - we won't change those APIs anymore after a year or so - and keeping the number of Beta APIs small will help in the API review negotiations.

Reply:
I pushed those changes to remove Beta tag - please re-review this PR